demos/gaze_estimation_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,117 @@
+    -o "<path>"              Optional. Name of output to save.
+    -limit "<num>"           Optional. Number of frames to store in output. If 0 is set, all frames are stored.
+    -res "<WxH>"             Optional. Set camera resolution in format WxH.
+    -m "<path>"              Required. Path to an .xml file with a trained Gaze Estimation model.
+    -m_fd "<path>"           Required. Path to an .xml file with a trained Face Detection model.
+    -m_hp "<path>"           Required. Path to an .xml file with a trained Head Pose Estimation model.
+    -m_lm "<path>"           Required. Path to an .xml file with a trained Facial Landmarks Estimation model.
+    -m_es "<path>"           Required. Path to an .xml file with a trained Open/Closed Eye Estimation model.
+    -d "<device>"            Optional. Target device for Gaze Estimation network (the list of available devices is shown below).
+    -d_fd "<device>"         Optional. Target device for Face Detection network (the list of available devices is shown below).
+    -d_hp "<device>"         Optional. Target device for Head Pose Estimation network (the list of available devices is shown below).
+    -d_lm "<device>"         Optional. Target device for Facial Landmarks Estimation network (the list of available devices is shown below).
+    -d_es "<device>"         Optional. Target device for Open/Closed Eye network (the list of available devices is shown below).
+    -fd_reshape              Optional. Reshape Face Detector network so that its input resolution has the same aspect ratio as the input frame.
+    -no_show                 Optional. Don't show output.
+    -r                       Optional. Output inference results as raw values.
+    -t                       Optional. Probability threshold for Face Detector. The default value is 0.5.
+    -u                       Optional. List of monitors to show initially.
+```
+
+Running the application with an empty list of options yields an error message.
+
+For example, to do inference on a CPU, run the following command:
+
+```sh
+./gaze_estimation_demo_gapi -d CPU -i <path_to_video>/input_video.mp4  -m <path_to_model>/gaze-estimation-adas-0002.xml -m_fd <path_to_model>/face-detection-retail-0004.xml -m_hp <path_to_model>/head-pose-estimation-adas-0001.xml -m_lm <path_to_model>/facial-landmarks-35-adas-0002.xml
+```
+
+### Run-Time Control Keys
+
+The demo allows you to control what information is displayed in run-time.
+The following keys are supported:
+
+* G - to toggle displaying gaze vector
+* B - to toggle displaying face detector bounding boxes
+* O - to toggle displaying head pose information
+* L - to toggle displaying facial landmarks
+* E - to toggle displaying eyes state
+* A - to switch on displaying all inference results
+* N - to switch off displaying all inference results
+* F - to flip frames horizontally
+* Esc - to quit the demo
+
+## Demo Output
```

Comment:
```suggestion
You can save processed results to a Motion JPEG AVI file or separate JPEG or PNG files using the -o option:

* To save processed results in an AVI file, specify the name of the output file with avi extension, for example: -o output.avi.
* To save processed results as images, specify the template name of the output image file with jpg or png extension, for example: -o output_%03d.jpg. The actual file names are constructed from the template at runtime by replacing regular expression %03d with the frame number, resulting in the following: output_000.jpg, output_001.jpg, and so on. To avoid disk space overrun in case of continuous input stream, like camera, you can limit the amount of data stored in the output file(s) with the limit option. The default value is 1000. To change it, you can apply the -limit N option, where N is the number of frames to store.
> **NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, OpenCV FFMPEG backend can be downloaded by the PowerShell script provided with the OpenVINO install package and located at <INSTALL_DIR>/opencv/ffmpeg-download.ps1. Run the script with Administrative privileges. Alternatively, you can save results as images.

## Demo Output
```

Reply:
unify with develop version of README